File: UnitTests/CoffeeMaker/CSHARP/CoffeeMaker/interfaces/ICoffee.cs

using System.Collections.Generic;

namespace CoffeeMaker.interfaces
{
    public interface ICoffee
    {
        public CoffeeModel AssignCoffeeProperties(int sugarLumps, string label, bool withMilk, string origin = null);

        public int GetBeans();


        public bool GrindBeans(string powerSource);


        public List<CoffeeModel> GroupOrder();


        public string dry_beans();
    }
}

File: UnitTests/CoffeeMaker/Java/src/main/java/Interfaces/ICoffee.java
package Interfaces;

import Models.CoffeeModel;

import java.io.IOException;
import java.util.List;

public interface ICoffee {

    public CoffeeModel assignCoffeeProperties(int sugarLumps, String label, Boolean withMilk, String origin);

    public int getBeans();

    public boolean grindBeans(String powerSource);

    public List<CoffeeModel> groupOrder();

    public String dry_beans();

    public boolean runGetCoffee()  throws IOException;
}
